src/style.css

```diff
@@ -0,0 +1,36 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  border-radius: 0.5px;
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 .stars__star {
+  background-image: url(./images/star.svg);
+}
```

Comment:
could you put this rule to other list when you set stars below

Reply:
yes i did it and this shortened the code